src/SignalR/clients/ts/signalr/src/DefaultHttpClient.ts

```diff
@@ -27,6 +30,7 @@ export class DefaultHttpClient extends HttpClient {
 
     /** @inheritDoc */
     public send(request: HttpRequest): Promise<HttpResponse> {
+        request.headers = {...this.headers, ...request.headers};
```

Comment:
I believe the order of these should be swapped. We would prefer user set headers to be applied over headers set by the library.
```typescript
// Apply user headers after library headers
request.headers = { ...request.headers, ...this.headers };
```